migrations/deploy_all.js

```diff
@@ -20,7 +20,7 @@ async function main() {
   await voteManagerMigration();
   await delegatorMigration();
   await faucetMigration();
-  await schellingCoinMigration();
+  //await schellingCoinMigration();
```

Comment:
@adi44 You can delete `Schelling coin` deployment here & also `9_delpoy_schelling_coin.js` file in `/migrations/src`.

Reply:
@dev1644 I have done the needful